example/src/tests.ts

```diff
@@ -672,6 +672,23 @@ test('canManagePreferences', async () => {
     )
   }
 
+  const boConsentList = await bo.contacts.consentList()
+  await delayToPropogate()
+
+  if (boConsentList.size !== 1) {
+    throw new Error(`consent list for bo should 1 not ${boConsentList.size}`)
+  }
+
+  const boConsentListState = boConsentList.get(
+    `address-${alixConversation.peerAddress.toLowerCase()}`
```

Comment:
The consent list were meant to be flexible allowing for things other than just addresses to be stored. For example blocking a `conversationId` or entire group chats etc... Because of this there is a bit of knowledge that needs to be understood about how we store the keys in the map. A prefix of address will match to this address being allowed or denied. cc @fabriguespe as this may be a good thing to document.